src/commands/registries/azure/DockerSiteCreateStep.ts

```diff
@@ -78,6 +79,38 @@ export class DockerSiteCreateStep extends AzureWizardExecuteStep<IAppServiceWiza
         }
     }
 
+    /**
+     * Has a few temporary workarounds so that the sdk allows some newer properties on the plan

```

Comment:
Please rephrase this comment so that it is clear what needs to be done and when.

1. If this method is to be entirely removed eventually, please say so. Under what circumstances can that happen?
2. If this method will stay, but portions ("temporary workarounds") are to be removed, please indicate which portion is to be removed, and under what circumstances.
3. What is "the sdk" mentioned here? azureextensionui? Some Azure SDK?

Reply:
Updated the comment. Hope this make sense now.

Reply:
Yup, much better! thank you!